source/turtles/turtle_hero.py

```diff
@@ -137,3 +160,47 @@ def jump(self, initial_v, grav_acc):
                 self.initial_y = 0
                 self.jump_counter = 0
                 print("end")
+
+    def update_anim_stop_rigth(self):
+        self.image = self.get_image(self.walk_r[1][0] * self.WIDTH, self.walk_r[1][1] * self.HEIGHT, self.WIDTH, self.HEIGHT)
```

Comment:
`self.image = self.get_image(self.walk_r[1][0] * self.WIDTH, self.walk_r[1][1] * self.HEIGHT, self.WIDTH, self.HEIGHT)`
In the update_anim_[..] functions, the only things that seem to change are the indices used to access `self.walk_r`. Would't it be much cleaner to make a separate function wrapping around `self.image = self.get_image(self.walk_r[1][0] * self.WIDTH, self.walk_r[1][1] * self.HEIGHT, self.WIDTH, self.HEIGHT)` and taking indices as arguments?